airflow/config_templates/default_airflow.cfg

```diff
@@ -581,6 +581,10 @@ authenticate = False
 # DAGs submitted manually in the web UI or with trigger_dag will still run.
 use_job_schedule = True
 
+# Allow externally triggered DagRuns for Execution Dates in the future when True
+# Only has effect if: use_job_schedule = False
+run_future_exec_dates = False
```

Comment:
Why is this necessary? Does it mean if a user wants to allow tasks on DagRuns with execution_date in the future to be executed, they have to turn off `use_job_schedule` globally?

>Only has effect if: use_job_schedule = False

Reply:
necessary otherwise cron sched tasks would run too early

Reply:
If you need to prevent DAGs with schedule_interval being cron expression from being run too early, why not check if the dag.schedule_interval is None? (instead of forcing `use_job_schedule` to be False globally) ?

That way you can allow both cron expression and externally triggered DAGs to co-exist.